src/main/java/pika/ui/TaskList.java

```diff
@@ -0,0 +1,148 @@
+package pika.ui;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import pika.exception.PikaException;
+import pika.task.Deadline;
+import pika.task.Event;
+import pika.task.Task;
+import pika.task.TaskTime;
+import pika.task.Todo;
+
+public class TaskList { //TaskList class used to store the tasks and will be updated from the Command class
+    protected int count;
+    private final ArrayList<Task> listArray;
```

Comment:
Perhaps `listArray` would be better named as `tasks`? In my opinion, this helps clarify it being a multi-valued variable as well.